src/commands/auth/__tests__/auth-command.utils.spec.ts

```
@@ -0,0 +1,41 @@
+      'https://cloud.revisium.io',
+    );
+  });
+
+  it('formats login hints for instance and URL targets', () => {
+    expect(
+      formatAuthLoginHint({
+        instanceName: 'cloud',
+        baseUrl: 'https://cloud.revisium.io',
+        credential: 'admin',
+      }),
+    ).toBe('--instance \'cloud\' --credential \'admin\' --api-key');
```

ESLint

[error] Replace `'--instance·\'cloud\'·--credential·\'admin\'·--api-key'` with `"--instance·'cloud'·--credential·'admin'·--api-key"`

(prettier/prettier)